<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConditionWidget</class>
 <widget class="QWidget" name="ConditionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget_condition">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>条件1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2"/>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="ConditionRelationShipConfigWidget" name="widget_conditionRelationShipConfig" native="true"/>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_confirm">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_addCondition">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/pic/add_64px.png</normaloff>:/pic/add_64px.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="ConditionConfigWidget" name="widget_conditionConfig" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConditionConfigWidget</class>
   <extends>QWidget</extends>
   <header>ConditionConfigWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConditionRelationShipConfigWidget</class>
   <extends>QWidget</extends>
   <header>ConditionRelationShipConfigWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
